bridge/scripts/mint-script.ts

```diff
@@ -0,0 +1,286 @@
+    const response = await safeService.confirmTransaction(
+        safeTxHash,
+        signature.data
+    );
+
+    console.log("Transaction confirmed:", response);
+    return { safeTxHash, confirmationResponse: response };
+}
+
+async function executeTransaction(
+    safeTxHash: string,
+    safeSdk: Safe = safeSdkOwner1
+) {
+    let safeBalance = await safeSdk.getBalance();
+
+    console.log(
+        `[Before Transaction] Safe Balance: ${ethers.utils.formatUnits(
+            safeBalance,
+            "ether"
+        )} ETH`
+    );
+
+    const safeTransaction = await safeService.getTransaction(safeTxHash);
+    const executeTxResponse = await safeSdk.executeTransaction(safeTransaction);
+    const receipt = await executeTxResponse.transactionResponse?.wait();
+
+    console.log("Transaction executed:");
+    console.log(
+        `${process.env.ETHERSCAN_ROOT_URL}/tx/${receipt?.transactionHash}`
+    );
+
+    safeBalance = await safeSdk.getBalance();
+
+    console.log(
+        `[After Transaction] Safe Balance: ${ethers.utils.formatUnits(
+            safeBalance,
+            "ether"
+        )} ETH`
+    );
+}
+
+function sleep(sec: number) {
+    return new Promise((resolve) => setTimeout(resolve, sec * 1000));
+}
+
+async function main(destination: string, amount: string) {
+    const amountToNum = Number(amount);
+
+    if (Number.isNaN(amountToNum)) {
+        console.log(`amount:${amount} is not a number`);
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    // Validation Check of Transfer Amount
+    if (
+        amountToNum > Number(process.env.MAX_TRANSFER_AMOUNT) ||
+        amountToNum === 0
+    ) {
+        console.log("Invalid Amount Input");
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    try {
+        console.log("destination??", destination);
+        ethers.utils.getAddress(destination);
+    } catch (e) {
+        console.error(e);
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    await sleep(1);
+    console.clear();
+
+    console.log(
+        "==========================================================================================================="
+    );
+    console.log(
+        `Are you trying to Transfer WNCG. To: ${destination}, Amount: ${amount} ??`
+    );
+    console.log('If Correct, Enter "yes", If not Enter anything');
+
+    const reader = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+    });
+
+    await reader.on("line", async (input) => {
+        reader.close();
+
+        if (input.toLowerCase() !== "yes") {
+            console.log("Cancel Minting....");
+            process.exit(1);
+        }
+
+        console.log(`Trnasfer WNCG to ${destination}, amount: ${amount} ...`);
+
+        await initalizeSafe();
+
+        const recipient = destination;
+        const gasPrice = 40;
```

Comment:
maybe change the gasPrice to 50, so that it could be more safe, based on : https://milkroad.com/ethereum/gas

Reply:
I think that it needed to be configurable (by command line args or env)

Reply:
@ahramyun7 @longfin 

I added optional CLI argument "gasPrice".

If it(=gasPrice) is not came from cli arguments, 
First: value of gasPrice is set to process.env.gasPrice
Second: If env variable (=gasPrice) is not exists, then value of gasPrice is set to default( =50 )